pubspec.yaml

```diff
@@ -2,7 +2,6 @@ name: pass_emploi_app
 description: Pass Emploi iOS and Android app.
 
 publish_to: 'none'
-version: 0.6.0+28
```

Comment:
Si on supprime le build-name (`0.6.0`) ici, on le sette jamais ? 
Du coup je crois que [la doc](https://docs.flutter.dev/deployment/android#updating-the-apps-version-number) dit que ça sera `1.0.0` par défaut ➡️  faudrait soit le laisser là soit le rajouter dans la commande `flutter build apk` de l'action, non ?